heim-host/src/sys/windows/users.rs

```diff
@@ -27,6 +31,44 @@ impl User {
         }))
     }
 
+    pub fn try_from_sid(sid: PSID) -> Result<Self> {
+        // name and domain cannot be longer than 256
+        let mut name_cch: DWORD = 256;
+        let mut name: Vec<WCHAR> = Vec::with_capacity(name_cch as usize);
+        let mut domain_cch: DWORD = 256;
+        let mut domain: Vec<WCHAR> = Vec::with_capacity(domain_cch as usize);
+
+        // winapi does not have SID_NAME_USE enum
+        let mut account_type: SID_NAME_USE = 0;
```

Comment:
I'm not sure if I understand this comment. Should it be [`SidTypeUser`](https://docs.rs/winapi/0.3.8/winapi/um/winnt/constant.SidTypeUser.html) in here?

Reply:
It's supposed to be an enum but winapi made it an `u32`